install/kots/Makefile

```diff
@@ -24,6 +24,31 @@ create_unstable_release:
 	replicated release create --lint --ensure-channel --yaml-dir ${YAML_DIR} --promote ${CHANNEL_UNSTABLE}
 .PHONY: create_unstable_release
 
+generate_preflight_checks:
```

Comment:
nit: Hmm. So Thinking more about this approach here i.e generating the kots-preflight pod manifest from the `installer-job` and changing it through `yq`, I was thinking its a bit too much but also writing a replica of `gitpod-installer-job` isn't a great idea too. So, Not too many opinions here. I think, we can move forward for now.

Reply:
The intention is that this goes away when Replicated does the [feature request](https://github.com/replicated-collab/replicated-gitpod/issues/51). I think the `yq` script is the least-bad way of doing it, but I'm very happy to be challenged on that

Reply:
> least-bad way of doing it,
Agree